pmd-cli/src/main/java/net/sourceforge/pmd/cli/commands/internal/DesignerCommand.java

```diff
@@ -25,17 +44,77 @@ public class DesignerCommand extends AbstractPmdSubcommand {
     @Override
     protected CliExitCode execute() {
         final String[] rawArgs = spec.commandLine().getParseResult().expandedArgs().toArray(new String[0]);
-        final ExitStatus status = DesignerStarter.launchGui(rawArgs);
 
-        return status == ExitStatus.OK ? CliExitCode.OK : CliExitCode.ERROR;
+        Path pmdDistDir = Paths.get(System.getenv("PMD_DIST_DIR"));
```

Comment:
rather than have an env variable, can't we get it at runtime with

```java
getClass().getProtectionDomain()
                    .getCodeSource()
                    .getLocation()
                    .toURI()
                    .getPath(); // path to pmd-cli.jar
```